model/span.go

```diff
@@ -30,20 +30,26 @@ type SpanModel struct {
 	Shared         bool              `json:"shared"`
 	LocalEndpoint  *Endpoint         `json:"localEndpoint,omitempty"`
 	RemoteEndpoint *Endpoint         `json:"remoteEndpoint,omitempty"`
-	Annotations    []Annotation      `json:"annotations"`
-	Tags           map[string]string `json:"tags"`
+	Annotations    []Annotation      `json:"annotations,omitempty"`
+	Tags           map[string]string `json:"tags,omitempty"`
 }
 
 // MarshalJSON exports our Model into the correct format for the Zipkin V2 API.
 func (s SpanModel) MarshalJSON() ([]byte, error) {
 	type Alias SpanModel
+
+	var timestamp uint64
+	if !s.Timestamp.IsZero() {
```

Comment:
The reason for this change is because on no timestamp we were marshalling it as `-6795364578871345`.

Reply:
good call... I've found some other issues in model as well which made sense to handle in #23 with unit tests added for checking these edge cases.